src/methods/method_overloading/hard/q4/RectangleOverload.java

```diff
@@ -0,0 +1,53 @@
+package methods.method_overloading.hard.q4;
+
+/*
+Write a Java Program that prompts the user to enter two integers and two doubles. Then write four overloaded methods
+called rectangleArea. The first takes both integer values for each side of the rectangle. The second takes the first
```

Comment:
We should probably add something about the goal of the question being to calculate the area of the rectangles based on the inputs (I feel like it is inferred, but not specifically stated.